lib/smart_answer_flows/marriage-abroad.rb

```diff
@@ -284,7 +284,7 @@ def define
 
       outcome :outcome_same_sex_civil_partnership_affirmation
 
-      outcome :outcome_os_marriage_impossible_no_laos_locals
+      outcome :outcome_opposite_sex_marriage_laos_with_no_lao_national
```

Comment:
I've gone with `outcome_opposite_sex_marriage_in_laos_without_lao_national ` in dfce5a818be2cf7369350ca3ba07f4f5ceaba9fb (PR #2409).